themes/SuiteR/include/MySugar/tpls/MySugar.tpl

```diff
@@ -156,7 +154,7 @@
 </div>
 <script type="text/javascript" src="include/MySugar/javascript/AddRemoveDashboardPages.js"></script>
 <script type="text/javascript" src="include/MySugar/javascript/retrievePage.js"></script>
-<link rel="stylesheet" type="text/css" href="themes/SuiteP/css/dashboardstyle.css">
+<link rel="stylesheet" type="text/css" href="themes/SuiteR/css/dashboardstyle.css">
```

Comment:
Suite7 doesn't use a custom MySmarty.tpl to customise the dashboard. It relies on the original MySugar.tpl found in include/MySugar/tpls/MySugar.tpl.